src/odemis/gui/model/__init__.py

```diff
@@ -561,7 +564,7 @@ def __init__(self, main):
         # See class docstring for more info.
         self.focussedView = VigilantAttribute(None)
 
-        layouts = {VIEW_LAYOUT_ONE, VIEW_LAYOUT_22, VIEW_LAYOUT_FULLSCREEN}
+        layouts = {VIEW_LAYOUT_ONE, VIEW_LAYOUT_22, VIEW_LAYOUT_VERTICAL, VIEW_LAYOUT_FULLSCREEN}
```

Comment:
Let's not change the possible layouts for all the tabs automatically. You can redefine the viewLayout in the EnzelAlignGUIData.